addons/website_sale/views/templates.xml

```diff
@@ -1148,6 +1151,29 @@
                                         </div>
                                         <div id="product_option_block" class="d-flex flex-wrap w-100"/>
                                     </div>
+                                    <div t-if="accordion_enabled" id="product_accordion"
+                                        class="o_accordion_not_initialized accordion accordion-flush mt-4"
+                                    >
+                                        <div t-if="accordion_custom_item" class="accordion-item">
+                                            <div class="accordion-header mb-0 h6">
```

Comment:
Enforce margin top as well.  Typography preferences are set at theme level by the user (we don't want a `24px` gap here, for example) 😉

```suggestion
                                            <div class="accordion-header my-0 h6">
```